backend/src/test/java/ai/verta/modeldb/ExperimentTest.java

```diff
@@ -2934,20 +2938,22 @@ public void deleteExperiments() {
     // For experimentRun1
     GetComments getCommentsRequest =
         GetComments.newBuilder().setEntityId(experimentRunList.get(0).getId()).build();
-    GetComments.Response getCommentsResponse =
-        commentServiceBlockingStub.getExperimentRunComments(getCommentsRequest);
-    LOGGER.info(
-        "experimentRun1 getExperimentRunComment Response : \n"
-            + getCommentsResponse.getCommentsList());
-    assertTrue(getCommentsResponse.getCommentsList().isEmpty());
+    GetComments.Response getCommentsResponse;
```

Comment:
post delete the auth check will fail and an error will be thrown instead of empty comments